src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -178,6 +179,7 @@ public SlimefunItem(Category category, ItemStack item, String id, RecipeType rec
 	 * @since 4.1.11
 	 */
 	public String getPermission() 			{		return permission;		}
+	public String[] getPermissionMessage()    {       return permissionMessage;       }
```

Comment:
This getter makes no sense if it just returns the same constant default value for each object.
You should read the value from config in .register()

Reply:
Honestly I am not sure why I put a getter